test/e2e/config/nutanix.yaml

```diff
@@ -233,26 +233,26 @@ variables:
   NUTANIX_PORT: "9440"
   NUTANIX_INSECURE: true
   NUTANIX_ADDITIONAL_TRUST_BUNDLE: ""
-  KUBERNETES_VERSION: "v1.29.2"
-  KUBERNETES_VERSION_v1_26: "v1.26.13"
-  KUBERNETES_VERSION_v1_27: "v1.27.9"
-  KUBERNETES_VERSION_v1_28: "v1.28.6"
-  KUBERNETES_VERSION_v1_29: "v1.29.2"
+  KUBERNETES_VERSION: "v1.29.4"
+  KUBERNETES_VERSION_v1_27: "v1.27.13"
+  KUBERNETES_VERSION_v1_28: "v1.28.9"
+  KUBERNETES_VERSION_v1_29: "v1.29.4"
+  KUBERNETES_VERSION_v1_30: "v1.30.0"
   NUTANIX_SSH_AUTHORIZED_KEY: ""
   CONTROL_PLANE_ENDPOINT_IP: ""
   CONTROL_PLANE_ENDPOINT_IP_V124: ""
   CONTROL_PLANE_MACHINE_COUNT: 3
   WORKER_MACHINE_COUNT: 3
   NUTANIX_PRISM_ELEMENT_CLUSTER_NAME: ""
-  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME: "ubuntu-2004-kube-v1.29.2.qcow2"
-  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME_v1_26: "ubuntu-2204-kube-v1.26.13.qcow2"
-  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME_v1_27: "ubuntu-2204-kube-v1.27.9.qcow2"
-  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME_v1_28: "ubuntu-2204-kube-v1.28.6.qcow2"
-  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME_v1_29: "ubuntu-2204-kube-v1.29.2.qcow2"
+  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME: "ubuntu-2004-kube-v1.29.4.qcow2"
+  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME_v1_27: "ubuntu-2204-kube-v1.27.13.qcow2"
+  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME_v1_28: "ubuntu-2204-kube-v1.28.9.qcow2"
+  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME_v1_29: "ubuntu-2204-kube-v1.29.4.qcow2"
+  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME_v1_30: "ubuntu-2204-kube-v1.30.0.qcow2"
   CONTROL_PLANE_MACHINE_TEMPLATE_UPGRADE_TO: "cluster-upgrade-conformance"
   WORKERS_MACHINE_TEMPLATE_UPGRADE_TO: "cluster-upgrade-conformance"
-  NUTANIX_MACHINE_TEMPLATE_IMAGE_UPGRADE_TO: "ubuntu-2204-kube-v1.29.2.qcow2"
-  NUTANIX_MACHINE_TEMPLATE_IMAGE_UPGRADE_FROM: "ubuntu-2204-kube-v1.28.7.qcow2"
+  NUTANIX_MACHINE_TEMPLATE_IMAGE_UPGRADE_TO: "ubuntu-2204-kube-v1.30.0.qcow2"
+  NUTANIX_MACHINE_TEMPLATE_IMAGE_UPGRADE_FROM: "ubuntu-2204-kube-v1.29.4.qcow2"
```

Comment:
It'd be good to think of some automation to keep this list up to date by creating PRs. similar to dependabot but for images.

Reply:
we can do that when we get them in CI runs. currently they all run manually